src/ctap/mod.rs

```diff
@@ -1094,9 +1097,43 @@ mod test {
                 // The expected response is split to only assert the non-random parts.
                 assert_eq!(fmt, "packed");
                 let mut expected_auth_data = vec![
-                    0xA3, 0x79, 0xA6, 0xF6, 0xEE, 0xAF, 0xB9, 0xA5, 0x5E, 0x37, 0x8C, 0x11, 0x80,
-                    0x34, 0xE2, 0x75, 0x1E, 0x68, 0x2F, 0xAB, 0x9F, 0x2D, 0x30, 0xAB, 0x13, 0xD2,
-                    0x12, 0x55, 0x86, 0xCE, 0x19, 0x47, 0x41, 0x00, 0x00, 0x00, 0x00,
+                    0xA3,
```

Comment:
This formatting is not really convenient.

Reply:
Since it's a test we can just do `expected_auth_data.push(INITIAL_SIGNATURE_COUNTER)` which should hopefully make cargo-fmt compress the rest.

Reply:
I didn't notice `cargo fmt` changed that. Changed to push the last byte, and fixed.